source/lambda/customResourceKendraDataSource/lambda_function.py

```diff
@@ -8,6 +8,19 @@ def on_create(event, context):
     dataBucketName = os.environ['DATA_BUCKET_NAME']
     roleArn = os.environ['KENDRA_ROLE_ARN']
 
+    create_faq_response = kendra_client.create_faq(
+        IndexId=kendraIndexId,
```

Comment:
Currently it seems we are creating faqs and syncing for the covid_faq.csv
Would it be possible to create 2 directories under the KendraPDFs folder - documents/ and faqs/ ?

We may  also need to update the s3 configuration on [datasource](https://github.com/awslabs/document-understanding-solution/pull/66/files#diff-0d95b0b8be64ff1927d3b5fa282113c6R30)

Reply:
Csv files are not indexed by Kendra (at this moment) and the pdfs synching was successful in my tests,  but yes, we can certainly reorganize the folders to make it neater.

Reply:
+1 Yep lets re-arrange the directory

I think the entire bucket gets synced.
It seems the entire bucket is [synced during the deployment ](https://github.com/awslabs/document-understanding-solution/blob/kendraMaster/source/lib/cdk-textract-stack.ts#L249) in the [custom resource](https://github.com/awslabs/document-understanding-solution/blob/kendraMaster/source/lambda/customResourceKendraDataSource/lambda_function.py#L17)